### PR TITLE
added space between mobile text and number

### LINKED ISSUE #248: Phone number needs space after call us
## Description
View the home page or call to action at bottom of a story. Originally reported in https://github.com/neontribe/ca-hospice-hope/issues/100.

## Steps to reproduce
1. On a wide viewport (intended for non-phone devices), view that phone number in homepage info section, or at the bottom of a story

Screenshot_20191002_170345

Screenshot_20191002_170323

### PR DESCRIPTION
**Pull request by SarikaWirtz**
Fixes #248

## Description

- Fixed space between mobile text and number
-

@neontribe/contemplating-action

### CODE REVIEW

**Review by HedgehogHolly — APPROVED** [review]
Looks good to me! 🐧

File: src/CallToAction.elm [hunk 1]
```diff
@@ -18,8 +18,11 @@ callToActionButton callToAction aClass =
                 [ getIcon callToAction.icon (Just "button--icon")
                 , span [] [ text callToAction.promptLong ]
                 ]
-            , span []
-                [ text callToAction.displayHref ]
+                ,
```
Comment (katjam) [inline src/CallToAction.elm]: I'm not sure about the formatting. If you have `elm-format` installed can you try running to see if it changes?

File: src/CallToAction.elm [hunk 2]
```diff
@@ -18,8 +18,11 @@ callToActionButton callToAction aClass =
                 [ getIcon callToAction.icon (Just "button--icon")
                 , span [] [ text callToAction.promptLong ]
```
Comment (katjam) [inline src/CallToAction.elm:19]: @Nikomus does it need to be in different spans for styling?
Or could go with a `, span [] [ text (callToAction.promptLong ++ " ") ]`

File: src/CallToAction.elm (hunk 1 above)
Comment (SarikaWirtz) [reply src/CallToAction.elm]: yes nick just told me about it. i will install now.

File: src/CallToAction.elm (hunk 2 above)
Comment (Nikomus) [reply src/CallToAction.elm:19]: It could be, but I think this is clear.

**Review by Nikomus — APPROVED** [review]
Nice one! 🛰

**Pull request merged**